pkg/auth/auth.go

```diff
@@ -34,8 +35,22 @@ func CheckAuthFile(authfile string) error {
 }
 
 // Login login to the server with creds from Stdin or CLI
-func Login(ctx context.Context, systemContext *types.SystemContext, opts *LoginOptions, registry string) error {
-	server := getRegistryName(registry)
+func Login(ctx context.Context, systemContext *types.SystemContext, opts *LoginOptions, registry []string) error {
```

Comment:
`registry []string` does not quite make sense. `args`?

Applies similarly to the logout code.